src/app/modules/main/chat_section/controller.nim

```diff
@@ -190,12 +202,6 @@ proc init*(self: Controller) =
 proc getMySectionId*(self: Controller): string =
   return self.sectionId
 
-proc getActiveChatId*(self: Controller): string =
```

Comment:
If you want, next time you can just do a forward declaration by putting `proc getActiveChatId*(self: Controller): string` above.